lib/parlour/conflict_resolver.rb

```diff
@@ -93,18 +100,6 @@ def resolve_conflicts(namespace, &resolver)
             next
           end
 
-          # Special case: are they all clearly equal? If so, remove all but one
-          if all_eql?(children)
-            Debugging.debug_puts(self, Debugging::Tree.end("All children are identical"))
-
-            # All of the children are the same, so this deletes all of them
-            namespace.children.delete(T.must(children.first))
-          
-            # Re-add one child
-            namespace.children << T.must(children.first)
-            next
-          end
-
```

Comment:
I think this block should still stay, since it handles other cases too.

Reply:
Thanks @paracycle; I was debating deleting it, glad you mentioned it, I will put it back.  I will also try adding a test for it since currently all the tests are passing without it.